src/redirects/ResetPasswordRedirect.tsx

```diff
@@ -0,0 +1,43 @@
+import { isNotDefined } from '@togglecorp/fujs';
+import {
+    Navigate,
+    generatePath,
+    useParams,
+} from 'react-router-dom';
+
+import { wrappedRoutes } from '../App/routes.tsx';
+
+interface ResetPasswordParams extends Record<string, string | undefined> {
+    uuid: string | undefined;
+    token: string | undefined;
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const { uuid, token } = useParams<ResetPasswordParams>();
+
+    const resetPasswordLink = (uuid && token) ? ({
+        pathName: (generatePath(wrappedRoutes.resetPassword.absolutePath)),
+        state: {
+            uuid,
+            token,
+        },
+    }) : ({
+        pathName: (generatePath(wrappedRoutes.fourHundredFour.absolutePath)),
+        state: {},
+    });
+
+    if (isNotDefined(resetPasswordLink)) {
+        return null;
+    }
+
+    return (
+        <Navigate
+            to="/reset-password"
```

Comment:
to should be used from wrappedRoutes.resetPassword as well.
Also, if pathName is not used in state, remove it and rename the link.